core/math/audio_frame.h

```diff
@@ -34,7 +34,7 @@
 #include "core/math/vector2.h"
 #include "core/typedefs.h"
 
-static inline float undenormalise(volatile float f) {
+static inline float undenormalize(volatile float f) {
```

Comment:
Stupid question: if `denormalize` denotes un-doing `normal`ization of audio, is `un-de-normalize` a double negation? If so, is it equivalent again to `normalize`?

If we have no one around to answer, we don't need to do anything for this PR, I'm just curious because as a non-audio person this one seems a bit weird-ish, and a brief Google didn't yield much, but 🤷 👀

Reply:
It's probably OK and referring to denormals (subnormal numbers). Seems like it's simply detecting denormals and reducing them to 0, I guess to avoid performance issues (not sure if it's relevant for modern CPUs). So it's not "normalize" but "get rid of denormals".